src/PhpSpec/Console/Formatter.php

```diff
@@ -49,6 +49,7 @@ public function __construct($decorated = false, array $styles = array())
         $this->setStyle('value', new OutputFormatterStyle('yellow'));
         $this->setStyle('lineno', new OutputFormatterStyle(null, 'black'));
         $this->setStyle('code', new OutputFormatterStyle('white'));
+        $this->setStyle('label', new OutputFormatterStyle('white', null, array('bold')));
```

Comment:
It's a good question, the answer is there are several things hard to read right now and it should be addressed in a different set of changes.